qa/common/check_copyright.py

```diff
@@ -40,6 +43,7 @@
 
 single_re = re.compile(COPYRIGHT_YEAR_RE0)
 range_re = re.compile(COPYRIGHT_YEAR_RE1)
+new_range_re = re.compile(COPYRIGHT_YEAR_RE2)
```

Comment:
call this `affiliate_re` or something relevant to the change

Reply:
Yes, updated.